libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -800,9 +800,17 @@ bool IRGeneratorForStatements::visit(BinaryOperation const& _binOp)
 	if (TokenTraits::isCompareOp(op))
 	{
 		if (auto type = dynamic_cast<FunctionType const*>(commonType))
+			// what's this common Type class ?
```

Comment:
https://github.com/ethereum/solidity/blob/6849774bd758d0e984ed676eb49ed47c38aec0e7/libsolidity/ast/ASTAnnotations.h#L309-L314

Reply:
https://github.com/ethereum/solidity/blob/6849774bd758d0e984ed676eb49ed47c38aec0e7/libsolidity/analysis/TypeChecker.cpp#L1704

Where it gets set.